A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_WHEEL.js

```diff
@@ -84,31 +95,31 @@ var A320_Neo_LowerECAM_WHEEL;
             if (!this.isInitialised) {
                 return;
             }
+            this.updateBrakeTemp(_deltaTime);
             this.updateTempColor(_deltaTime);
             this.updateHydroSymbols(_deltaTime);
             this.updateAutoBrake(_deltaTime);
             this.updateSkidToggle(_deltaTime);
             this.updateSpoilerSpeedBrake(_deltaTime);
             this.updateLandingGear(_deltaTime);
-            this.updateBrakeTemp(_deltaTime);
         }
 
         updateTempColor(_deltaTime) {
-            let max = this.CurrentBrakeTemps[0];
+            let max = this.currentBrakeTemps[0];
             let maxIndex = 0;
 
-            for (var i = 1; i < this.CurrentBrakeTemps.length; i++) {
-                if (this.CurrentBrakeTemps[i] > max) {
+            for (let i = 1; i < this.currentBrakeTemps.length; i++) {
+                if (this.currentBrakeTemps[i] > max) {
                     maxIndex = i;
-                    max = this.CurrentBrakeTemps[i];
+                    max = this.currentBrakeTemps[i];
                 }
             }
 
-            for (var i = 0; i < this.BrakeTempsText.length; i++) {
+            for (let i = 0; i < this.brakeTempsText.length; i++) {
                 if (i === maxIndex && max > 300) {
```

Comment:
```suggestion
                if (this.currentBrakeTemps[i] > 300) {
```
According to komp on the discord, every brake temp above 300°C should be amber. The maxIndex can still be used for the "hottest brake arc", if you intend to add that later.

Reply:
I will make sure both of those are implemented.